packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -297,4 +301,76 @@ class WKWebView {
   Future<void> loadRequest(NSUrlRequest request) {
     throw UnimplementedError();
   }
+
+  /// Loads the contents of the specified HTML string and navigates to it.
+  Future<void> loadHtmlString(String string, String? baseUrl) {
+    throw UnimplementedError();
+  }
+
+  /// Loads the web content from the specified file and navigates to it.
+  Future<void> loadFileUrl(String url, String readAccessUrl) {
+    throw UnimplementedError();
+  }
+
+  /// Loads the Flutter asset specified in the pubspec.yaml file.
+  Future<void> loadFlutterAsset(String key) {
```

Comment:
This is not a method provided by `WKWebView`, but adding it to our subclass seems to be the easiest solution: https://github.com/flutter/plugins/blob/main/packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m#L258

Reply:
Agreed. Maybe explicitly mention in the comment that this is a utility method implemented on the native side rather than an actual WKWebView method. Probably pretty clear from the name, but it doesn't hurt to be extra clear.